koch.nim

```diff
@@ -477,10 +459,6 @@ proc runCI(cmd: string) =
   ## build nimble early on to enable remainder to depend on it if needed
   kochExecFold("Build Nimble", "nimble")
 
-  when false:
```

Comment:
please keep this and maybe add a comment `# consider re-enabling` or similar; original PR https://github.com/nim-lang/Nim/pull/10150 showed `when not defined(windows):` and then was turned into `when false` in https://github.com/nim-lang/Nim/commit/c5dbb0379fc431a01220224097f00323df6c9ced ; as a result compile time FFI is not being tested in testament anymore and it started to bitrot, cf my PR from yesterday https://github.com/nim-lang/Nim/pull/11991
CTFFI is a super useful feature that I heavily rely on